app/lib/constants.rb

```diff
@@ -0,0 +1,277 @@
+module Constants
+  ALLOWED_MASTODON_INSTANCES = [
```

Comment:
So we have a file similar to this `app/lib/constants/mastodon.rb` now, that I think was added after you made this PR. We can probably remove this constant, and the `SiteConfig` constant.